app/models/user.rb

```diff
@@ -2,11 +2,15 @@
 # A User can have many posts
 # It also has a name, bio, image, which are required fields.
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] as part of the requirements The application should ask for a confirmation email. kindly fix it by adding `:confirmable` 🙏